jupyter-extension/jupyterlab_pachyderm/handlers.py

```diff
@@ -444,16 +445,29 @@ async def put(self, path):
 
 
 def setup_handlers(web_app):
+    """
+    Sets up the Pachyderm client and the HTTP request handler.
+
+    Config for the Pachyderm client will first be attempted by reading
+    the local config file. This falls back to the PACHD_ADDRESS and
+    DEX_TOKEN env vars, and finally defaulting to a localhost client
+    on the default port 30650 failing that.
+    """
     try:
         client = Client().from_config()
         get_logger().debug(
             f"Created Pachyderm client for {client.address} from local config"
         )
     except FileNotFoundError:
-        client = Client()
-        get_logger().debug(
-            "Could not find config file, creating localhost Pachyderm client"
-        )
+        if PACHD_ADDRESS:
```

Comment:
I'm unsure if `PACHD_ADDRESS` will always be present in determined notebook workflow, will test